admin_guide/docker_registry.adoc

```diff
@@ -0,0 +1,158 @@
+/registry/docker/registry/v2/repositories/p1/pause/_uploads <7>
+/registry/docker/registry/v2/repositories/p1/pause/_layers <8>
+/registry/docker/registry/v2/repositories/p1/pause/_layers/sha256
+/registry/docker/registry/v2/repositories/p1/pause/_layers/sha256/a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4
+/registry/docker/registry/v2/repositories/p1/pause/_layers/sha256/a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4/link <9>
+/registry/docker/registry/v2/repositories/p1/pause/_layers/sha256/f72a00a23f01987b42cb26f259582bb33502bdb0fcf5011e03c60577c4284845
+/registry/docker/registry/v2/repositories/p1/pause/_layers/sha256/f72a00a23f01987b42cb26f259582bb33502bdb0fcf5011e03c60577c4284845/link
+----
+<1> This directory stores all layers and signatures as blobs
+<2> This file contains the blob's contents
+<3> This directory stores all the image repositories
+<4> This directory is for a single image repository `p1/pause`
+<5> This directory contains signatures for a particular image manifest revision
+<6> This file contains a reference back to a blob (which contains the signature data)
+<7> This directory contains any layers that are currently being uploaded and staged for the given repository
+<8> This directory contains links to all the layers this repository references
+<9> This file contains a reference to a specific layer that has been linked into this repository via an image
+
+== Pruning
+
+=== Overview
+
+To prevent unbounded growth of images and layers, it is possible for cluster administrators to run a command to prune old, unused images and layers. 
+
+=== Pruning Algorithm
+
+OpenShift uses the following logic to determine which images and layers to prune:
+
+Remove any image managed by OpenShift (see below) that was created at least *n* minutes ago and is *not* currently referenced by:
+
+- any pod created less than *n* minutes ago
+- any image stream created less than *n* minutes ago
+- any running pods
+- any pending pods
+- any replication controllers
+- any deployment configs
+- any build configs
+- any builds
+- the 3 most recent items in stream.status.tags[].items (3 is configurable)
+
+There is no support for pruning from external registries.
+
+Images "managed by OpenShift" will have an annotation indicating this fact. The current proposed annotation is openshift.io/image.managed.
+
+The minimum age criterion should default to 60 minutes.
+
+When removing an image, remove all references to the image from all ImageStreams having a reference to the image in `status.tags`.
+
+Also automatically remove any image layer that is no longer referenced by any images.
+
+=== Running `osadm prune-images`
+
+The command to prune images and layers is `osadm prune-images`. It supports the following options:
+
+- `--dry-run` - displays what images and layers would be pruned, but does not perform any real deletions. Defaults to `true`.
```

Comment:
You have to use `--dry-run=false`, yes. The other prune subcommands (builds, deployments) default to true.